cfgov/v1/tests/models/test_home_page.py

```diff
@@ -17,3 +17,12 @@ def test_preview_default_template(self):
 
     def test_preview_2021_template(self):
         self.check_preview_template('home_page_2021', 'v1/home_page_2021.html')
+
+    def test_highlight_card_block_rendering(self):
+        block = HighlightCardBlock()
+        value = block.to_python({
+            'heading': 'Highlight',
```

Comment:
Should you include a test without a heading?

Reply:
See above, the block requires a heading, even though the card template makes it optional.